migrations/2018_01_30_220100_create_posts_users_table.php

```diff
@@ -18,7 +18,6 @@
             $table->integer('post_id')->unsigned();
             $table->integer('user_id')->unsigned();
 
-            $table->foreign('post_id')->references('id')->on('posts')->onDelete('cascade');
```

Comment:
#1339 will change the posts table to InnoDB (https://github.com/flarum/core/pull/1339/commits/d4850eccb7b6c5bb95bb2090f032250f24e88b25), so this can be re-added with a migration that has a more recent date/time.

Reply:
I think this migration is already newer than yours:

```
migrations/2018_01_11_120604_change_posts_table_to_innodb.php
```